integration-box/csv_import_via_http/http_csv_download_sample.dig

```diff
@@ -0,0 +1,16 @@
+timezone: "Asia/Tokyo"
+
+schedule:
+ daily>: 07:45:00
+
++task1:
+  py>: csv_import_td.main
+  url           : https://raw.githubusercontent.com/waysaku/treasure-boxes/csv_import_via_http/integration-box/csv_import_via_http/sample_data/sample_member.csv
+  database      : DATABASE NAME
+  table         : TABLE NAME
```

Comment:
[nits] Maybe, it'd be better to have some example database and table name so that customers can run this example after pulling the repo like:

```
database: csv_import
table: replaced_table
```